tests/controller/webapp/test_rap_views.py

```diff
@@ -847,13 +847,15 @@ def test_status_view_tracing_with_unexpected_rap_ids(db, client, monkeypatch):
     job1 = job_factory(action="action1", backend="test")
     job2 = job_factory(action="action1", backend="test")
     job3 = job_factory(action="action1", backend="test1")
+    job_factory(action="action1", backend="test1")
```

Comment:
should be backend `test` - currently is the same as `job3`

```suggestion
    job_factory(action="action1", backend="test")
```

Reply:
oh yes, sorry!